src/Microsoft.AspNetCore.Server.Kestrel.Core/CoreStrings.resx

```diff
@@ -0,0 +1,123 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Now that we have this, I think we should move all our strings to this file (not in this PR of course). Can you file a task for that?